src/main/webapp/web.xml

```diff
@@ -16,6 +16,24 @@
         <url-pattern>/</url-pattern>
     </servlet-mapping>
 
+    <servlet>
+        <servlet-name>loginController</servlet-name>
+        <servlet-class>taxi.controller.LoginController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>loginController</servlet-name>
+        <url-pattern>/login</url-pattern>
+    </servlet-mapping>
```

Comment:
I prefer to use annotations such as WebServlet and WebFilter, but it's up to you)

Reply:
Pas de probleme 👌